tinytuya/Cloud.py

```diff
@@ -127,7 +127,7 @@ def setregion(self, apiRegion=None):
         if self.apiRegion == "in":
             self.urlhost = "openapi.tuyain.com"      # India Datacenter
 
-    def _tuyaplatform(self, uri, action='GET', post=None, ver='v1.0', recursive=False, query=None, content_type='application/json'):
+    def _tuyaplatform(self, uri, action='GET', post=None, ver='v1.0', recursive=False, query=None, content_type=None):

```

Comment:
Agree with this. It could be considered a breaking change for the API, but since this is a _function() we aren't breaking the contract.